go.mod

```diff
@@ -274,5 +274,6 @@ replace (
 	// fix potential security issue(CVE-2020-26160) introduced by indirect dependency.
 	github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.6-0.20210809144907-32ab6a8243d7+incompatible
 	github.com/pingcap/tidb/parser => ./parser
+	github.com/tikv/pd/client => github.com/ti-chi-bot/pd/client v0.0.0-20230209034200-6d23a31c24be
```

Comment:
Why does it change to use ```ti-chi-bot/pd```?

@nolouch

Reply:
I will change it after https://github.com/tikv/pd/pull/5981 merged

Reply:
I hope it can be merged after https://github.com/tikv/pd/pull/5981 merged